test/unit/providers/boleto-api-caixa/index.js

```diff
@@ -178,8 +182,15 @@ test('buildPayload complete', async (t) => {
         maxDaysToPayPastDue: 60,
       },
     },
+    payeeGuarantor: {
+      name: `${boleto.company_name}`,
```

Comment:
opa, aqui como não estamos concatenando ou montando uma string dinâmica, podemos passar a variável direto 😀 

```suggestion
      name: 'boleto.company_name',
```

Reply:
feito